src/lava/magma/core/model/py/ports.py

```diff
@@ -260,6 +530,59 @@ def write(self, data: int, idx: int):
 
 class PyVarPort(AbstractPyPort):
     """Python implementation of VarPort used within AbstractPyProcessModel.
+
+    A PyVarPort is a Port related to a variable Var of a Process and might be
+    connected to a RefPort of another process. It is used to get or set the
+    value of the referenced Var across Processes. A PyVarPort is connected via
+    two channels to a PyRefPort. One channel is used to send data from the
+    PyRefPort to the PyVarPort and the other is used to receive data from the
+    PyVarPort. PyVarPorts set or send the value of the related Var (service())
+    given the command VarPortCmd received by a connected PyRefPort.
```

Comment:
Here and below, I would replace "related var" -> "linked var".

Reply:
Done